src/base/net/smtp.cpp

```diff
@@ -138,7 +138,7 @@ void Smtp::sendMail(const QString &from, const QString &to, const QString &subje
 {
     const Preferences *const pref = Preferences::instance();
     m_message = "Date: " + getCurrentDateTime().toLatin1() + "\r\n"
-                + encodeMimeHeader(u"From"_qs, from)
+                + encodeMimeHeader(u"From"_qs, tr("qBittorrent %1").arg(u'<' + from + u'>'))
```

Comment:
Is "qBittorrent" intended to be translated?

Reply:
It was already done in some translations, I did it for consistency reasons with the existing logic:

https://github.com/qbittorrent/qBittorrent/blob/cbf9c8ec1a9f734e9cf950b6bf4c62b24cb0ebc4/src/lang/qbittorrent_ar.ts#L1092
https://github.com/qbittorrent/qBittorrent/blob/cbf9c8ec1a9f734e9cf950b6bf4c62b24cb0ebc4/src/lang/qbittorrent_fa.ts#L1092
https://github.com/qbittorrent/qBittorrent/blob/cbf9c8ec1a9f734e9cf950b6bf4c62b24cb0ebc4/src/lang/qbittorrent_hi_IN.ts#L1092

But they are few - only 3 of them, so there is an option 2 - should we consider do not translate this line as (1) there are few existing translations and (2) qBittorrent is the official name of the product and it likely should not be translatable?

Reply:
> qBittorrent is the official name of the product and it likely should not be translatable

👍

Reply:
Done in https://github.com/qbittorrent/qBittorrent/pull/17374/commits/79df93d83803e75a27b06e0715cdb304b8268f6a